### PR TITLE
feat: update documentation

### LINKED ISSUE #75: Add GitHub issue templates

### LINKED ISSUE #76: Describe what the project does in the README

### LINKED ISSUE #77: Add PR template

### LINKED ISSUE #78: Add a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by sanchegm**
All below issues relate to [Issue #21 ](https://github.com/ClinGen/stanford-affils/issues/21).

- Create Issue and Bug templates. Relates to [Issue #75]( https://github.com/ClinGen/stanford-affils/issues/75). Closes  #75

- Update README.md, consolidate information, rename files. Relates to [Issue #76](https://github.com/ClinGen/stanford-affils/issues/76). Closes  #76

- Add PR template. Relates to [Issue #77](https://github.com/ClinGen/stanford-affils/issues/77). Closes #77 

- Add contributing file. Relates to [Issue #78](https://github.com/ClinGen/stanford-affils/issues/78). Closes #78

### CODE REVIEW

**Comment by sanchegm** [comment]
This is a big PR, next time I will break these down into smaller PRs.

**Review by liammulh — COMMENTED** [review]
This is looking really good, thank you @sanchegm! I really like using issue templates and PR templates.

The reason for the `doc` directory structure was that we are trying to follow a specific documentation system (the [Divio documentation system](https://docs.divio.com/documentation-system/)) where you make a distinction between beginner-friendly tutorials, step-by-step how-to guides for people who have the requisite background knowledge, and long-form explanations (e.g. why some architectural decision was made), hence having those files. I'm not opposed to having individual files for things, but it would be nice if we could continue to use the Divio system since that's what we're using now in gci-vci-aws and in the HCI repo. But if you have strong feelings that it should be done a different way, I'm open to new and better ways of doing things.

Also, it looks like some of the files exceed 80 characters, which I think the auto-formatter should have caught.

File: doc/explanation.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
 # Explanations
 
+Description from
```
Comment (liammulh) [inline doc/explanation.md:3]: I like the idea of linking to the Divio description! 👍🏻

File: README.md [hunk 2]
```diff
@@ -1,8 +1,40 @@
-# affils
+# ClinGen Affiliation Service
+
+This service provides a way for ClinGen users to interact (CRUD) with the
```
[the remark this answers is not in the document]
Comment (sanchegm) [reply README.md:3]: This looks great! I have made this change.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,40 @@
...
+  - Running [code checks](./doc/how-to.md#run-code-checks)
+  - Expectations on [writing new code](./doc/how-to.md#write-code-that-can-be-submitted-to-the-main-branch)
+    to be submitted
+  - [Organizing imports and constants](./doc/how-to.md#organize-imports-and-constants)
+
+## Getting Started and Tutorials
+
+- [Tutorials](./tutorials.md#get-started). Guides to getting set up and other
+  learning oriented tasks.
+- Setting up [environment variables](./how-to.md#environment-variables)
```
[the remark this answers is not in the document]
Comment (sanchegm) [reply README.md:26]: Added!

File: doc/how-to.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+How to create environment variaables. There's a `.env.template` file that is
+the source of truth for environment variable keys.
+
+- Create a copy of `.env.template` and name it `.env`
+- Do not rename any of keys for the envornment variables.
+- Change the path for `AFFILS_DB_FILE` and placeholder values.
+
+## Contributing
+
+How to contribute to the Affiliations Serive.
```
[the remark this answers is not in the document]
Comment (sanchegm) [reply doc/how-to.md:76]: Fixed!

File: doc/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please describe your changes and the related issue or bug.
+
+## Issue ticket number and link
+
+Please include the issue ticket number and link. This project only accepts pull requests related to open issues. Add `closes #XXXX` in your comment to auto-close the issue that this PR fixes.
+
+## Checklist
```
[the remark this answers is not in the document]
Comment (sanchegm) [reply doc/pull_request_template.md:9]: Should we keep it as is, then update after you merge your PR?

File: doc/reference.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+## References
```
[the remark this answers is not in the document]
Comment (sanchegm) [reply doc/reference.md:1]: Okay! I have removed it!

**Pull request merged**